src/docs/product/performance/mobile-vitals.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Mobile Vitals"
+sidebar_order: 60
+description: "Learn more about Mobile Vitals in Sentry, and how they give you a better understanding of your mobile application's health."
+---
+
+Mobile Vitals are a set of metrics to measure how fast your app starts, the duration of HTTP requests, the number of slow and frozen frames, and more. Each metric provides insights into the overall performance of your mobile application. These metrics give you the details you need to not only prioritize critical performance issues, but to trace issues down to the root cause to solve them faster.
+
+Mobile Vitals are summarized in several graphs on the **Performance** page in [sentry.io](https://sentry.io) when you select "Mobile" in the "Display" dropdown.
+
+![Performance page in Sentry with Mobile Vitals.](mobile-vitals.png)
+
+## App Start
+
+App start metrics track how long your mobile application takes to launch. For this, Sentry measures *cold starts* and *warm starts*.
+
+- **Cold Start**: A cold start refers to when the app launches for the first time, after a reboot or update. The app is not in memory and no process exists.
+- **Warm Start**: A warm start refers to when the app has already launched at least once and is partially in memory. For instance, the user backs out of your app, but then re-launches it. The process may have continued to run, but the app must recreate the activity from scratch.
+
+On iOS, Apple recommends your app take at most 400ms to render the first frame. On Android, the [Google Play console](https://developer.android.com/topic/performance/vitals/launch-time#av) warns you when a cold start takes longer than five seconds or a warm start longer than two seconds. The exact definitions differ slightly per platform. Learn more in our [iOS](/platforms/apple/guides/ios/performance/instrumentation/automatic-instrumentation/) and [Android](/platforms/android/performance/instrumentation/automatic-instrumentation/) SDK docs.
```

Comment:
Do we have a reference for that 400ms on iOS? And I was kind of expecting to see mentions of those thresholds (400ms on iOS and 5s on Android) within each of the platform's documentation as well but I'm not seeing it.

Reply:
We don't display any warnings or anything yet. These are just recommendations of Apple and Google.

Reply:
If we are to consolidate them into Mobile Vitals, is it possible for us to pick some reasonable defaults for mobile as a whole so that we are able to do things like display warnings?

Reply:
Is this a blocker for publishing these docs once the rest of the review is complete?

Reply:
Not a blocker 👍